packages/adapter-netlify/index.js

```diff
@@ -26,111 +46,29 @@ export default function ({ split = false } = {}) {
 			const publish = get_publish_directory(netlify_config, builder) || 'build';
 
 			// empty out existing build directories
-			builder.rimraf(publish);
+			builder.rimraf('.netlify/edge-functions');
 			builder.rimraf('.netlify/functions-internal');
 			builder.rimraf('.netlify/server');
 			builder.rimraf('.netlify/package.json');
 			builder.rimraf('.netlify/handler.js');
 
-			builder.mkdirp('.netlify/functions-internal');
-
 			builder.log.minor(`Publishing to "${publish}"`);
 
-			builder.writeServer('.netlify/server');
-
 			// for esbuild, use ESM
 			// for zip-it-and-ship-it, use CJS until https://github.com/netlify/zip-it-and-ship-it/issues/750
 			const esm = netlify_config?.functions?.node_bundler === 'esbuild';
 
-			/** @type {string[]} */
-			const redirects = [];
-
-			const replace = {
-				'0SERVER': './server/index.js' // digit prefix prevents CJS build from using this as a variable name, which would also get replaced
-			};
-
-			if (esm) {
-				builder.copy(`${files}/esm`, '.netlify', { replace });
+			if (edge) {
+				await generate_edge_functions({ builder });
```

Comment:
It looks like `generate_edge_functions` doesn't support function splitting — is it worth making this clear if someone tries to use both together?

```suggestion
			if (edge) {
				if (split) {
					throw new Error('Cannot use `split: true` alongside `edge: true`');
				}
				
				await generate_edge_functions({ builder });
```

Reply:
Ah, good point!